charts/rancher-vsphere-csi/Chart.yaml

```diff
@@ -1,11 +1,11 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: vSphere CSI
+  catalog.cattle.io/kube-version: '>= 1.19.0-0 < 1.23.0-0'
```

Comment:
This should be `< 1.24.0-0` to match CPI since we are live with 1.23 support

Reply:
per our call a few minutes ago, the kube-version will not be modified in this PR